src/server/api/controllers/organization.controller.js

```diff
@@ -0,0 +1,88 @@
+"use strict";
+
+const HttpError = require("../lib/utils/http-error");
+const knex = require("../../config/db");
+
+const createOrganization = async ({ body }) => {
+  const { name, logo } = body;
+  const organization = await knex
+    .from("organizations")
+    .select("*")
+    .where({
+      name
+    });
+  if (organization.length !== 0) {
+    throw new HttpError("Bad request", "organization already exists!", 409);
+  }
+  return knex("organizations").insert({
+    name: name,
+    logo: logo
```

Comment:
Just a heads up. You have a few places where you are doing assignments like this. Could be optimized with destructuring like this:
```
{ name, logo }
```
But it's a really minor improvement.